composer-scripts/post-create-project.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+use craft\helpers\Console;
+use craft\helpers\StringHelper;
```

Comment:
Realized I could use these Craft helpers, which makes `ScriptHelpers` super minimal (only need it for file text replacements).